nasa-gibs/worldview
# wv-2702 orbit tracks

## Description
We developed the associated layers section for orbit tracks before we had layers that were created from multiple satellites traveling in the same orbit direction and time of day. Now that we have a few layers like that, HLS Sentinel-2A & B and Landsat 8 & 9 and OPERA Surface Water Extent, we should consider including the satellite name as well so you can discern which orbit track is for what satellite.

Fixes # wv-2702.

## How To Test

- git checkout wv-2702-orbit-tracks
- npm ci
- npm run watch
- Load a layer with multiple satellites (such as "OPERA Dynamic Surface Water Extent Provisional (L3)" or "Reflectance (Nadir BRDF-Adjusted)") and a layer with a single satellite (such as 'Deep Blue Aerosol Type')
- Click the 'View Options' icon to open the options menu
- Confirm under "Associated Layers" section the text label includes the satellite name
- Enable an orbit track by clicking on the checkbox(es) in "Associated Layers"
- Confirm that the associated orbit track is listed within the layer row & that this text also includes the satellite name

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
